tests/files/gce_oel-aio-flannel-kubeadm-sep.yml

```diff
@@ -0,0 +1,13 @@
+# Instance settings
+cloud_image_family: oracle-7
```

Comment:
@ant31 added. This line is just a stub for now - we will have to add the Oracle 7 image to our GCE account. It is not one of the public images available on GCE.

Reply:
how can it be tested then ?

Reply:
I'm hoping we can add the Oracle 7 image to the GCE account that is used for this project

Reply:
how to do that?

Reply:
@ant31 I've got an Oracle 7.5 image packaged, and GCE ready here:

https://storage.googleapis.com/born-images/oracle75/compressed-image.tar.gz

Does this project have storage available in Google Storage? If so, it can be copied there, then referenced when creating a new image @ 

https://console.cloud.google.com/compute/imagesAdd?project=<kubespray project>

Set family name to oracle-7, and we should be good to go.

Reply:
Now that we are migrating away from GCE, what matters is the packet_ CI job.